<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
 background-color:rgb(255, 253, 225);

}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>847</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,2">
       <item>
        <widget class="QLabel" name="label_iconousuario">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_18">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_exito">
           <property name="styleSheet">
            <string notr="true">QLabel {
               font: 12pt;
               color: green;
               }</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_nombreusuario">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
border-radius: 6px;
border: 1px solid rgb(166,164, 180);
background-color:white;
color: rgb(103,80,41);
padding: 5px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Usuario...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_contrasenausuario">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
border-radius: 6px;
border: 1px solid rgb(166,164, 180);
background-color:white;
color: rgb(103,80,41);
padding: 5px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Contrase√±a del usuario...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_roles">
           <property name="styleSheet">
            <string notr="true">QComboBox {
border-radius: 6px;
border: 1px solid rgb(166,164, 180);
background-color:white;
color: rgb(103,80,41);
padding: 5px;
}</string>
           </property>
           <item>
            <property name="text">
             <string>Administrador</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Empleado</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Proyectista</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Armadores</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Otro</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_19">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_error">
           <property name="styleSheet">
            <string notr="true">QLabel {
               font: 12pt;
               color: red;
               }</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_20">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_tabla">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(141, 110, 58);</string>
         </property>
         <property name="text">
          <string>Tabla</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_acceso">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(141, 110, 58);</string>
         </property>
         <property name="text">
          <string>Acceso a</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="tablaslist">
         <property name="styleSheet">
          <string notr="true">QComboBox{
border: 1px solid #95753d;
background-color: white;
font: 75 14pt ;
color: rgb(149, 117, 61);
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_acciones">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(141, 110, 58);</string>
         </property>
         <property name="text">
          <string>Acciones</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="accioneslist">
         <property name="styleSheet">
          <string notr="true">QComboBox{
border: 1px solid #95753d;
background-color: white;
font: 75 14pt ;
color: rgb(149, 117, 61);
}</string>
         </property>
         <item>
          <property name="text">
           <string>Ver</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Escritura</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,0,0">
       <item>
        <spacer name="verticalSpacer_17">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancelar">
         <property name="styleSheet">
          <string notr="true">QPushButton {
border-radius: 6px;
border: 3px solid #95753d;
background-color: #95753d;
color: white;
height: 25px;
}

QPushButton:hover {
	background-color: rgb(116, 91, 47);
	color: rgb(255, 255, 255);
	border: 3px solid  rgb(116, 91, 47);
}

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(103, 80, 41);
	color: rgb(255, 255, 255);
	border: 3px solid  rgb(103, 80, 41);
}</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_confirmar">
         <property name="styleSheet">
          <string notr="true">QPushButton {
border-radius: 6px;
border: 3px solid #95753d;
background-color: #95753d;
color: white;
height: 25px;
}

QPushButton:hover {
	background-color: rgb(116, 91, 47);
	color: rgb(255, 255, 255);
	border: 3px solid  rgb(116, 91, 47);
}

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(103, 80, 41);
	color: rgb(255, 255, 255);
	border: 3px solid  rgb(103, 80, 41);
}</string>
         </property>
         <property name="text">
          <string>Confirmar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_16">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>847</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
